### DISCUSSION TITLE
How to nest existing styles?

### DISCUSSION THREAD

**Discussion by adaboese** [chakra-ui/panda#1812] [Q&A]
```ts
const prose = css({
  '& > * + *': {
    marginTop: '16px',
  },
});

const theme = css({
  '& .milkdown-aimd-theme': {
    // ... I need prose styles here
  },
});
```

**Comment by segunadebayo** [comment]
We documented this in the merging styles docs. Please see here: https://panda-css.com/docs/concepts/merging-styles#merging-css-objects

The solution is to switch to `css.raw`, which is an identity function that preserves the object.

```jsx
// preserve style object
const proseStyles = css.raw({
  '& > * + *': {
    marginTop: '16px',
  },
});

// get the classname for previous usage
const prose = css(proseStyles)

// use the style object here
const theme = css({
  '& .milkdown-aimd-theme': {
    ...proseStyles
  },
});
```

Here's a working example: https://stackblitz.com/edit/vitejs-vite-r84b2t?file=src%2FApp.tsx&terminal=dev

**Reply by adaboese** [reply]
Thank you!

**Reply by adaboese** [reply]
Hm. Does not appear to work. The styles are not extracted.

**Reply by adaboese** [reply]
Looks like `css.raw` and the `css` to which it is inlined have to be in the same file. Not ideal, but I made it work!

**Reply by segunadebayo** [reply]
Yes indeed, it needs to be  in the same file for the compiler to be able to “evaluate” the code.

We’ll document this

**Reply by segunadebayo** [reply]
Another approach is to move this to a recipe or text style in Panda.
